dev/io.openliberty.security.jakartasec.3.0.internal_fat.config/test-applications/Discovery.war/src/discovery/servlets/DiscoveryServlet.java

```diff
@@ -0,0 +1,152 @@
+            updatedDiscData.add("userinfo_endpoint", "https://localhost:8920/oidc/endpoint/OP1/userinfo");
+            updatedDiscData.add("registration_endpoint", "https://localhost:8920/oidc/endpoint/OP1/registration");
+            updatedDiscData.add("scopes_supported", Json.createArrayBuilder().add("openid").add("general").add("profile").add("email").add("address").add("phone"));
+            updatedDiscData.add("claims_supported",
+                                Json.createArrayBuilder().add("sub").add("groupIds").add("name").add("preferred_username").add("picture").add("locale").add("email").add("profile"));
+            updatedDiscData.add("response_modes_supported", Json.createArrayBuilder().add("query").add("fragment").add("form_post"));
+            updatedDiscData.add("grant_types_supported",
+                                Json.createArrayBuilder().add("authorization_code").add("implicit").add("refresh_token").add("client_credentials").add("password").add("urn:ietf:params:oauth:grant-type:jwt-bearer"));
+            updatedDiscData.add("token_endpoint_auth_methods_supported", Json.createArrayBuilder().add("client_secret_post").add("client_secret_basic"));
+            updatedDiscData.add("display_values_supported", Json.createArrayBuilder().add("page"));
+            updatedDiscData.add("claim_types_supported", Json.createArrayBuilder().add("normal"));
+            updatedDiscData.add("claims_parameter_supported", false);
+            updatedDiscData.add("request_parameter_supported", false);
+            updatedDiscData.add("request_uri_parameter_supported", false);
+            updatedDiscData.add("require_request_uri_registration", false);
+            updatedDiscData.add("check_session_iframe", "https://localhost:8920/oidc/endpoint/OP1/check_session_iframe");
+            updatedDiscData.add("end_session_endpoint", "https://localhost:8920/oidc/endpoint/OP1/end_session");
+            updatedDiscData.add("revocation_endpoint", "https://localhost:8920/oidc/endpoint/OP1/revoke");
+            updatedDiscData.add("app_passwords_endpoint", "https://localhost:8920/oidc/endpoint/OP1/app-passwords");
+            updatedDiscData.add("app_tokens_endpoint", "https://localhost:8920/oidc/endpoint/OP1/app-tokens");
+            updatedDiscData.add("personal_token_mgmt_endpoint", "https://localhost:8920/oidc/endpoint/OP1/personalTokenManagement");
+            updatedDiscData.add("users_token_mgmt_endpoint", "https://localhost:8920/oidc/endpoint/OP1/usersTokenManagement");
+            updatedDiscData.add("client_mgmt_endpoint", "https://localhost:8920/oidc/endpoint/OP1/clientManagement");
+            updatedDiscData.add("code_challenge_methods_supported", Json.createArrayBuilder().add("plain").add("S256"));
+            updatedDiscData.add("backchannel_logout_supported", true);
+            updatedDiscData.add("backchannel_logout_session_supported", true);
+            updatedresponse = updatedDiscData.build();
```

Comment:
We're not using that method - I had kept that code in place in case we ever needed it, but, that will probably not be the case.  So, I've removed the entire method.